internal/dataprovider/providers/cloud/data_provider.go

```diff
@@ -39,7 +39,7 @@ const (
 type Identity struct {
 	Provider         string
 	Account          string
-	AccountAlias     string
+	AccountAlias     *string
```

Comment:
Would there be any downside if we avoided the pointer here? Instead of `nil` just use `""`.

Reply:
None that I can think of tbh. Maybe that's my java minded approach here. Absence of a value, to me, is nil. But no harm on keeping it empty, I believe.